.github/workflows/kcp-registrar-image.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
           containerfiles: |
             ./images/kcp-registrar/Dockerfile
           build-args: |
-            KCP_BRANCH=release-0.5
+            KCP_BRANCH=v0.6.0-alpha.0
```

Comment:
so far we have been following a release branch, rather than using a tagged version. We may want to change that in the future but I think that we should keep it this way for now, hence: release-0.6

Reply:
The release was just tagged this morning, hence why I was using the alpha version up to this point. Fixed.